cli/src/main/java/org/jboss/as/cli/impl/ReadlineConsole.java

```diff
@@ -862,7 +1053,7 @@ private void loop() {
                             }
                         }
                     });
-                }, completions, preProcessors, readlineHistory, null, READLINE_FLAGS);
+                }, completions, preProcessors, readlineHistory, cursorListener, READLINE_FLAGS);
```

Comment:
If char highlighting is disabled, the cursorListener could be null, aesh would then not call into the listener.

Reply:
OK. I thought a null matcher would be a better way to implement but I see your point